uniffi/huffman/README.md

```diff
@@ -0,0 +1,27 @@
+# libtw2_huffman
+
+Python bindings for huffman.
```

Comment:
What is "huffman"? This summary doesn't really tell me anything that I can't tell from the directory listing.